README.md

```diff
@@ -53,5 +53,15 @@ Timeseries like data will be in e.g. "HTR.csv"
 
 CCD-metadata will both be in a json per image and for all images in the "CCD.csv"
 
+### Downloading files from AWS
+
+With the aws-cli installed and the credetials file in place, you can easily download files or entire projects on the commandline. The following example downloads the project "MyArchive1_20200511" from S3 to the local directory "test"
+
+```aws s3 cp s3://mats-l0-artifacts/MyArchive1_20200511 ./test --recursive```
```

Comment:
```suggestion
```aws s3 cp s3://mats-l0-artifacts/binning2002/MyArchive1_20200511 ./test --recursive```
```
Since uploaded files are in projects / "folders" better have this in the example.